cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -187,16 +187,17 @@ def create_org(self):
             "alias": sarge.shell_format(' -a "{0!s}"', self.sfdx_alias)
             if self.sfdx_alias
             else "",
-            "email": sarge.shell_format('adminEmail="{0!s}"', self.email_address)
+            "email": sarge.shell_format(' adminEmail="{0!s}"', self.email_address)
             if self.email_address and not org_def_has_email
             else "",
             "default": " -s" if self.default else "",
+            "instance": f" instance={self.instance}" if self.instance else "",
             "extraargs": os.environ.get("SFDX_ORG_CREATE_ARGS", ""),
```

Comment:
I don't know of any uses cases we've applied for this parameter other than instance. I checked and SFDX will throw an error if `instance` should be specified twice (if we had a MetaCI global override as well as an org-level override). Perhaps ignore `instance` if the environment variable is non-empty, or if they both contain `instance`?

Reply:
Ah, good point. Or we could add a new environment variable for specifying the instance, so that we can do something like:
    
    instance = self.instance or os.environ.get("SFDX_SIGNUP_INSTANCE")

But really I think we probably want to get to a place where the instance can be configured in the db without restarting MetaCI, and so always passed into CumulusCI as part of the org config.

Reply:
@davidmreed I updated it so that it uses self.instance with a fallback to the SFDX_SIGNUP_INSTANCE env var